src/features/editor/components/animate-workspace/motion-preset-thumbnail.css

```
@@ -0,0 +1,203 @@
+  height: 28px;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.mp-shape {
+  height: 13px;
+  width: 13px;
+  border-radius: 3px;
+  background: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)